homeassistant/components/sensor/usps.py

```diff
@@ -60,6 +60,7 @@ def __init__(self, session, name):
         self._name = name
         self._attributes = None
         self._state = None
+        self.update()
```

Comment:
Ok, I think there's a race condition in certain cases between `get_packages` and `get_mail`. Both try to login at the same time, and therefore both fail. That's why doing *another* `update()` fixes it. I'm going to push a new version of `myusps` that exposes a `login()` method that we should call in `setup` to ensure we're logged in when HASS starts.